packages/charts/src/charts/bar/bar-chart.component.tsx

```diff
@@ -197,6 +201,7 @@ export const BarChart: FC<Props> = ({
                   groupMode={groupMode}
                   selectors={tooltip.selectors}
                   isList={tooltip.selectors.length > 1}
+                  formatTooltip={formatTooltip}
```

Comment:
IMO the _formatTooltip_ property should be named in that convention only at chart root level. For tooltip components I suggest to rename property to`formatValue`